hooks/README.md

```diff
@@ -91,7 +89,7 @@ React hook for getting device-class whether it's light or heavy
 import { useDeviceClass } from './device-class';
 
 const MyComponent = () => {
-  const deviceClass = useDeviceClass();
+  const { deviceClass } = useDeviceClass();
   return (
     <div>
```

Comment:
Pattern usage revisions to this point are 👍

Reply:
Thanks.